ui/pom.xml

```diff
@@ -38,17 +37,13 @@
         <configuration>
           <filesets>
             <fileset>
-              <directory>packages/models</directory>
+              <directory>.</directory>
               <includes>
-                <include>packages/models/dist/**</include>
-                <include>packages/models/node_modules/**</include>
-                <include>packages/ui/dist/**</include>
-                <include>packages/ui/node_modules/**</include>
-                <include>packages/services/dist/**</include>
-                <include>packages/services/node_modules/**</include>
-                <include>node/**</include>
-                <include>node_modules/**</include>
+                <include>**/dist/</include>
               </includes>
+              <excludes>
+                <exclude>**/node_modules/</exclude>
```

Comment:
@uidoyen @mdrillin this should be the fix for the second time build issue. There are many `/dist/` folders inside `/node_modules/` that were removed, but should be ignored.

Reply:
looks good to me